bndtools.core/src/bndtools/explorer/Model.java

```diff
@@ -116,7 +116,7 @@ private void update0() {
 		try {
 			// coalesce some more updates on
 			// the worker thread(s).
-			Thread.sleep(10);
+			Thread.sleep(50);
```

Comment:
If you want to wait, maybe `Display#timerExec` would be more appropriate (e.g. Line 108), blocking in the UI thread is bad...

Reply:
I know but this is intentional to give whoever is messing with SWT to finish. 50 ms is not yet noticeable. But I agree it is horrible, just lost for other options .... It is a an SWT problem but they do not want to look into it.

Reply:
> It is a an SWT problem but they do not want to look into it.

Can you link the [SWT issue](https://github.com/eclipse-platform/eclipse.platform.swt/issues)? Is there a reproducer?